### PR TITLE
Port of PDFTransform from sdx-transform-cs

### PR DESCRIPTION
**Pull request by warrenbailey**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,5 +3,6 @@ Changelog
 
 Unreleased
 ----------
+- Replacing PDF transformer with updated one from SDX-transform-cs
```
Comment (necrophonic) [inline CHANGELOG.rst]: Strictly speaking should be (I think the word is) imperative tense - "Replace" rather than "Replacing"

File: sdx/common/transforms/PDFTransformer.py [hunk 2]
```diff
@@ -50,15 +69,25 @@ def __init__(self, survey, response_data):
...
+
+        os.makedirs("./tmp/%s" % rndm_name)
+
+        tmp_name = "./tmp/%s/%s.pdf" % (rndm_name, rndm_name)
+        doc = SimpleDocTemplate(tmp_name, pagesize=A4)
+        doc.build(self.get_elements())
+
+        return os.path.realpath(tmp_name)
+
+    def get_elements(self):
```
Comment (tundish) [inline sdx/common/transforms/PDFTransformer.py]: This function should be a static method. It is stateless and needs only _survey_ and _response_.
Remove the references to self.

Codacy will mark you down otherwise ;-)

File: CHANGELOG.rst (hunk 1 above)
Comment (warrenbailey) [inline CHANGELOG.rst]: Done

File: sdx/common/transforms/test/test_image_transformer.py [hunk 3]
```diff
@@ -56,7 +56,7 @@ def test_mwss_index(self):
             # Build PDF
             fp = os.path.join(locn, "pages.pdf")
             doc = SimpleDocTemplate(fp, pagesize=A4)
-            doc.build(PDFTransformer.get_elements(survey, reply))
+            doc.build(PDFTransformer(survey, reply).get_elements())
```
Comment (tundish) [inline sdx/common/transforms/test/test_image_transformer.py]: Revert this change once get_elements is a static method.

File: sdx/common/transforms/PDFTransformer.py [hunk 4]
```diff
@@ -50,15 +69,25 @@ def __init__(self, survey, response_data):
     def render(self):
         buffer = BytesIO()
         doc = SimpleDocTemplate(buffer, pagesize=A4)
-        doc.build(self.get_elements(self.survey, self.response))
+        doc.build(self.get_elements())
```
Comment (tundish) [inline sdx/common/transforms/PDFTransformer.py]: Revert this change once get_elements is a static method.

File: sdx/common/transformer.py [hunk 5]
```diff
@@ -135,7 +135,7 @@ def pack(self, settings=None, img_seq=None, tmp="tmp"):
             # Build PDF
             fp = os.path.join(locn, "pages.pdf")
             doc = SimpleDocTemplate(fp, pagesize=A4)
-            doc.build(PDFTransformer.get_elements(survey, self.response))
+            doc.build(PDFTransformer(survey, self.response).get_elements())
```
Comment (tundish) [inline sdx/common/transformer.py]: Revert this change once get_elements is a static method.

**Comment by warrenbailey** [comment]
I've made all the requested changes
